engine/src/main/java/org/terasology/config/PermissionConfig.java

```diff
@@ -61,4 +64,20 @@ private static String createRandomKey() {
         }
         return codeBuilder.toString();
     }
-}
+
+    /**
+     * @return Returns the one-time auth key
+     */
+    public String getOneTimeAuthorizationKey() {
+        return oneTimeAuthorizationKey;
+    }
+
+    /**
+     * @param oneTimeAuthorizationKey Sets the one-time authorization key to this String
+     */
+    public void setOneTimeAuthorizationKey(String oneTimeAuthorizationKey) {
+        this.oneTimeAuthorizationKey = oneTimeAuthorizationKey;
+    }
+
+
+}
```

Comment:
I'm assuming you mean to revert only the EOF character rather than the whole change?